frontend/appflowy_tauri/src/appflowy_app/components/layout/NavigationPanel/NewFolderButton.tsx

```diff
@@ -17,7 +17,7 @@ export const NewFolderButton = ({ scrollDown }: { scrollDown: () => void }) => {
           <AddSvg></AddSvg>
         </div>
       </div>
-      <span>New Folder</span>
+      <span>New View</span>
```

Comment:
Maybe the filename should be NewViewButton and NewViewButton.hooks.ts

Reply:
done